mender-update/standalone/standalone.cpp

```diff
@@ -417,19 +422,81 @@ ResultAndError DoInstallStates(
 		return {Result::FailedNothingDone, payload.error()};
 	}
 
+	auto default_paths {main_context.GetConfig().paths};
+	events::EventLoop loop;
+	auto script_runner {executor::ScriptRunner(
+		loop,
+		chrono::seconds {main_context.GetConfig().state_script_timeout_seconds},
+		default_paths.GetArtScriptsPath(),
+		default_paths.GetRootfsScriptsPath())};
+
+
+	// Download Enter
+	{
+		auto err = script_runner.RunScripts(executor::State::Download, executor::Action::Enter);
+		if (err != error::NoError) {
+			err = err.FollowedBy(
+				script_runner.RunScripts(executor::State::Download, executor::Action::Error));
+			err = err.FollowedBy(update_module.Cleanup());
+			err = err.FollowedBy(RemoveStateData(main_context.GetMenderStoreDB()));
+			return {Result::FailedNothingDone, err};
+		}
+	}
```

Comment:
Any reason for the blocks? I think reusing `err` is safe and fine.

Reply:
I just thought it scoped it better